src/git-ship

```diff
@@ -34,9 +36,17 @@ function steps {
   fi
 
   echo "checkout_main_branch"
-  echo "fetch"
+
```

Comment:
Please remove the added newline. These two steps are closely related so I'd prefer keeping this right next to each other. Same for the other newlines added before the other `if [ "$has_remote" = true ]; then` statements